src/celengine/nebula.h

```diff
@@ -24,6 +24,18 @@ namespace celestia::util
 class AssociativeArray;
 }
 
+enum class NebulaType
```

Comment:
return this back to `Nebula` but rename to `Type`, and then use `Nebula::Type` instead of `NebulaType`

Reply:
NebulaTypeName, NebulaTypeNames, NebulaType have same structure as galaxy versions. Multiple calling functions are virtual and cross deepskyobjects, nebula, galaxy, globular and opencluster.  Keep present code and you can decide on preferred code and implement across all DSOs to maintain uniformity in the future.

Reply:
your reply is not related to the requested. `NebulaType was defined inside `Nebula`, please restore this. But rename `NebulaType` to `Type`